.cirrus.yml

```diff
@@ -147,6 +147,10 @@ task:
     - python test\util\test_runner.py
     - python test\util\rpcauth-test.py
   functional_tests_script:
+    # Increase the dynamic port range to the maximum allowed value to mitigate "OSError: [WinError 10048] Only one usage of each socket address (protocol/network address/port) is normally permitted".
+    # See: https://docs.microsoft.com/en-us/biztalk/technical-guides/settings-that-can-be-modified-to-improve-network-performance
+    - netsh int ipv4 set dynamicport tcp start=1025 num=64511
+    - netsh int ipv6 set dynamicport tcp start=1025 num=64511
```

Comment:
does this mean they might collide with the p2p and rpc ports from `[TEST_RUNNER_PORT_MIN, TEST_RUNNER_PORT_MIN + 2 * range]` (`[14000, 24000]`)?

Reply:
Yes, if an ephemeral port by accident belongs to that range a collision is possible. But this range limits only ports of P2P and RPC connections. Our test framework uses ports out of this range in the `feature_bind_extra.py` and `feature_proxy.py`. The former is skipped on Windows, but the latter still could be a source of port collisions.